aiven/resource_vpc_peering_connection.go

```diff
@@ -262,6 +269,12 @@ func copyVPCPeeringConnectionPropertiesFromAPIResponseToTerraform(
 		return err
 	}
 
+	if len(peeringConnection.UserPeerNetworkCIDRs) > 0 {
+		if err := d.Set("user_peer_network_cidrs", peeringConnection.UserPeerNetworkCIDRs); err != nil {
+			return err
+		}
+	}
```

Comment:
Should this not clear old cidrs in case list is empty?

Reply:
Indeed, I was trying here to distinguish between Gateway Attachment and regular VPC by checking this parameter empty, but it introduces this problem.